packages/web/src/pages/App.jsx

```diff
@@ -761,13 +756,6 @@ class App extends Component {
                   isMobile={isMobileClient}
                   component={AudioTransactionsPage}
                 />
-                <Route exact path={PURCHASES_PAGE} component={PurchasesPage} />
```

Comment:
out of scope for this PR, but just a thought - potentially could have the pay and earn page still respect and update routes like `/pay-and-earn/withdrawals`

Reply:
hmm good call. made a ticket.